Makefile

```
@@ -12,7 +12,8 @@ GOPROXY?=http://proxy.golang.org
 export GO111MODULE
 export GOPROXY
 
-PKGS=$(shell go list ./... | grep -v -E '/vendor/|/test|/examples')
+# go pakages for unit tests, excluding e2e tests
+PKGS_UT=$(shell go list ./... | grep -v /test/e2e)
```

lets keep the previous variable name

```suggestion
PKGS=$(shell go list ./... | grep -v /test/e2e)
```

---

corrected now :)